Problem-1.py

```diff
@@ -0,0 +1,32 @@
+
+#Approach
+# Every time we will choose or not choose the element if we choose same index will be passed to next
+#recursive call else pass the next index to next recursive call and find the min.
+
+
+# complexities
```

Comment:
This complexity is for recursive solution, but you submitted dp solution. Can you please check the complexity for both the questions